i18n/en.yaml

```diff
@@ -1,7 +1,7 @@
 readMore:
   other: Read More
 readingTime:
-  other: '{{ .ReadingTime }} min read'
+  other: '{{ .Count }} mins read'
```

Comment:
Maybe we can keep the `one` part and have something like `one: '1 min read'`?

Reply:
There you go